spec/controllers/hq/partners_controller_spec.rb

```diff
@@ -16,4 +16,32 @@ def foobar_action
       get :foobar_action
     end
   end
+
+  describe 'while signed-in' do
+    before :each do
+      sign_in instance_double(AdminUser)
+    end
+
+    describe 'GET #show' do
+      before :each do
+        @id= FactoryGirl.create(:partner).id
+      end
+
+      describe 'redirect & flash' do
+        example 'with invalid :id' do
+          get :show, id: 0
```

Comment:
Then perhaps `''` would have been better to use.  These tests are gone now anyway due to code review.